src/index.js

```diff
@@ -49,6 +52,74 @@ const EmojiPicker = ({
     []
   );
 
+  useEffect(() => {
+    let unsubscribe = tinykeys(window, {
+      ArrowUp: keyEvent => handleKeyboard(keyEvent.key),
+      ArrowDown: keyEvent => handleKeyboard(keyEvent.key),
+      ArrowLeft: keyEvent => handleKeyboard(keyEvent.key),
+      ArrowRight: keyEvent => handleKeyboard(keyEvent.key),
+    });
+    return () => {
+      unsubscribe();
+    };
+  }, []);
+
+  const handleKeyboard = key => {
+    const activeElementClass = document.activeElement.getAttribute('class');
+
+    if (activeElementClass && activeElementClass.startsWith('icn-')) {
+      handleCategoriesKeyboard(key);
+    } else if (activeElementClass === 'emoji-search') {
+      handSearchKeyboard(key);
+    } else {
+      handleEmojiesKeyboard(key);
+    }
+  };
+
+  const handleCategoriesKeyboard = key => {
+    switch (key) {
+      case ArrowRight:
+        document.activeElement.nextElementSibling.focus();
+        break;
+      case ArrowLeft:
+        document.activeElement.previousElementSibling.focus();
+        break;
+      case ArrowDown:
+        const searchInput = document.getElementsByClassName('emoji-search')[0];
+        searchInput.focus();
+        break;
+    }
+  };
+
+  const handSearchKeyboard = key => {
+    if (key === ArrowUp) {
+      const emojiCategories = document.getElementsByClassName(
+        'emoji-categories'
+      )[0];
+      emojiCategories.firstChild.focus();
+    }
+    if (key === ArrowDown) {
+      const firstEmoji = document.getElementsByClassName('emoji')[0];
+      firstEmoji.firstChild.focus();
+    }
+  };
+
+  const handleEmojiesKeyboard = key => {
```

Comment:
This doesn't handle the scenario of moving either back to the previous emoji group or the next. It also doesn't handle arrow up and down, so there's still quite some work to do here before we can ship this. But we can do it in a follow up PR